src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/CategoryFilter.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace Pim\Bundle\CatalogBundle\Elasticsearch\Filter;
```

Comment:
I'm wondering if we should not put our classes under the `Product` namespace
as we have some chances in a near future to index other entities

don't do that in this PR
but think about it to know if it's relevant or not